libp2p/NodeTable.cpp

```diff
@@ -299,6 +299,8 @@ void NodeTable::ping(NodeEntry const& _nodeEntry, boost::optional<NodeID> const&
         m_socket->send(p);
 
         m_sentPings[_nodeEntry.id] = {chrono::steady_clock::now(), pingHash, _replacementNodeID};
+        if (m_nodeEventHandler && _replacementNodeID)
```

Comment:
```suggestion
        if (_replacementNodeID && m_nodeEventHandler)
```
Check `_replacementNodeID` first as it was already accessed.